command/connect/envoy/envoy.go

```diff
@@ -110,6 +111,9 @@ func (c *cmd) init() {
 		"Set the agent's gRPC address and port (in http(s)://host:port format). "+
 			"Alternatively, you can specify CONSUL_GRPC_ADDR in ENV.")
 
+	c.flags.StringVar(&c.envoyVersion, "envoy-version", "1.13.0",
```

Comment:
Can we add this location to the checklist in `contributing` since it will need to be updated with each new version from now on?

Reply:
✅